### PR TITLE
LOK-2106: Fix headings, discovery issues, notifications comp

### PR DESCRIPTION
**Pull request by mikewrosey**
## Description
Changes the page headings to headline1. Minor adjustment to the map & layout to accommodate.
screenshot-localhost_3009-2023 10 10-11_16_38

Fix dark mode text for no discoveries. Uses the `disabled` color, as requested by ux.
screenshot-localhost_3009-2023 10 10-11_07_58

Re-adds this outbound notifications under component under a new name specific to locations. 
screenshot-localhost_3009-2023 10 10-11_19_39

Also found/removed some more unused code from the old `Appliances` page that we will not use.

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2106

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: ui/src/components/Locations/LocationsNotificationsCtrl.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<template>
+  <FeatherButton 
+    text
+    data-test="notifications-btn"
+    @click="openModal"
+  >
+    Outbound Notification
```
Comment (mikewrosey) [inline ui/src/components/Locations/LocationsNotificationsCtrl.vue:7]: This is the only comp from the old Appliances page that we need to keep. This was previously removed but re-added here under a new name `LocationsNotificationCtrl.vue`

File: ui/src/components/Locations/LocationsNotificationsCtrl.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const config: PagerDutyConfigInput = reactive({
+  integrationkey: undefined
+})
+const save = async () => {
+  await notificationMutations.savePagerDutyIntegrationKey({ config })
+  if (!notificationMutations.error) {
+    config.integrationkey = undefined
+    closeModal()
+    showSnackbar({
+      msg: 'Integration key successfuly saved.'
```
Comment (synqotik) [inline ui/src/components/Locations/LocationsNotificationsCtrl.vue:58]: should be: `successfully`

**Review by synqotik — APPROVED** [review]
LGTM, just one typo.

File: ui/src/components/Locations/LocationsNotificationsCtrl.vue (hunk 2 above)
Comment (mikewrosey) [reply ui/src/components/Locations/LocationsNotificationsCtrl.vue]: fixed!

**Pull request merged**